export const main = async (event: any) => {
    throw new Error('step-error');
}